docs/topics/class-based-views/generic-editing.txt

```diff
@@ -37,7 +37,7 @@ The view can be constructed using a ``FormView``:
 .. code-block:: python
     :caption: ``views.py``
 
-    from myapp.forms import ContactForm
+    from .forms import ContactForm
```

Comment:
This should be sorted after the django import with a blank line in between (apply this suggestion throughout).

Reply:
The rule is the following:
`from django import something`
_blank line_
`from .forms import something`
_blank line_
_blank line_
`code`

Is that correct? I found sometimes only one line between imports and [code](https://github.com/django/django/pull/17748/files#diff-37eaa1c6dcfe12b4514c34328ece5d1da0ce1380a461d2d9656294d5314a18a1L290). 

Do I have to follow this rule In this scenario or just switched the import is sufficient?

Reply:
Correct. I wouldn't worry about adding a second blank line between imports and code if there's only one.

Reply:
Alright, thank you very much. I'm ready to submit another PR.